packages/vscode-extension/src/debug/taskTerminal/ngrokTunnelTaskTerminal.ts

```diff
@@ -57,6 +57,10 @@ export interface INgrokTunnelArgs extends IBaseTunnelArgs {
   ngrokArgs: string[];
   ngrokPath?: string;
   tunnelInspection?: string;
+  output?: {
```

Comment:
`writeToEnvironmentFile`?

Reply:
The ngrok output change will be in another PR.